BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/AzureStorageAccount/AzureStorageAccountServiceCollectionExtensions.cs

```diff
@@ -20,6 +23,11 @@ public static void AddAzureStorageAccount(this IServiceCollection services, Azur
         services.Configure<AzureStorageAccountOptions>(opt => opt.ConnectionString = options.ConnectionString);
         services.AddSingleton<AzureStorageAccountContainerClientFactory>();
         services.AddScoped<IBlobStorage, AzureStorageAccount>();
+        services.AddSingleton(_ => new BlobServiceClient(options.ConnectionString));
```

Comment:
Use the extension method provided by the blob storage library instead

```suggestion
        services.AddBlobServiceClient(options.ConnectionString);
```

Reply:
This doesn't work. The reason: `The type 'Microsoft.Extensions.DependencyInjection.IServiceCollection' must be convertible to 'Azure.Core.Extensions.IAzureClientFactoryBuilder'`. The method is generic and needs an Azure specific builder. Looking inside the documentation, it tells me to use
```c#
services.AddAzureClients(cb => cb.AddBlobServiceClient(options.ConnectionString))
```
but this method doesn't exist either.

Reply:
Shouldn't matter anyway if we go with the approach I described in https://github.com/nmshd/backbone/pull/902#discussion_r1794689509.